addon/lib/utils/register-helper.js

```diff
@@ -0,0 +1,35 @@
+import Ember from 'ember';
+
+function registerHelperIteration1(name, helperFunction) {
+  //earlier versions of ember with htmlbars used this
+  Ember.HTMLBars.helpers[name] = helperFunction;
```

Comment:
Well I firstly tried to basically use this code in list-view, but it did not work, I encountered `Assertion Failed: makeBoundHelper generated helpers do not support use with blocks`. So I guess we can't just share it :(. Still I can at least add the missing `Ember.HTMLBars.helpers`case.